docs/configuration.rst

```diff
@@ -288,14 +288,16 @@ If you use this way to configure custom fragment types, ensure there is no ``too
 Each table within this array has the following mandatory keys:
 
 
-``directory``
-    The type / category of the fragment.
-
-``name``
+``name`` (required)
     The description of the fragment type, as it must be included
     in the news file.
 
-``showcontent``
+``directory`` (optional)
```

Comment:
As long as it has a default value, I think that you can infere that it's optinal.

It make sense to highlight the values that are required.


```suggestion
``directory``
```